caravel/assets/visualizations/mapbox.jsx

```diff
@@ -0,0 +1,336 @@
+// JS
+var d3 = window.d3 || require('d3');
+
+// CSS
+require('./mapbox.css');
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ScatterPlotOverlay from 'react-map-gl/src/overlays/scatterplot.react.js';
+import Immutable from 'immutable';
+import supercluster from 'supercluster';
+import ViewportMercator from 'viewport-mercator-project';
+
+const earthCircumferenceKm = 40075.16;
+
+class ScatterPlotGlowOverlay extends ScatterPlotOverlay {
+  _kmToPixels(kilometers, latitude, zoomLevel) {
+    // Algorithm from: http://wiki.openstreetmap.org/wiki/Zoom_levels
+    var latitudeRad = latitude * (Math.PI / 180);
+    // Seems like the zoomLevel is off by one
+    var kmPerPixel = earthCircumferenceKm * Math.cos(latitudeRad) / Math.pow(2, zoomLevel + 8 + 1);
+    return d3.round(kilometers / kmPerPixel, 2);
+  }
+
+  _isNumeric(num) {
+    return !isNaN(parseFloat(num)) && isFinite(num);
+  }
+
+  _drawText(ctx, pixel, fontHeight, label, radius, rgb, shadow) {
+    var maxWidth = radius * 2 * 0.9;
+    // Formula: https://en.wikipedia.org/wiki/Relative_luminance
+    var luminance = 0.2126*rgb[1] + 0.7152*rgb[2] + 0.0722*rgb[3];
+
+    ctx.globalCompositeOperation = "source-over";
```

Comment:
i don't think we are enforcing style via a linter yet, but we should stick to the airbnb js styleguide for all new js we write. the styleguide says we prefer `'` over `"` in js. 

https://github.com/airbnb/javascript#strings--quotes